freqtrade/rpc/telegram.py

```diff
@@ -148,7 +148,7 @@ def send_msg(self, msg: Dict[str, Any]) -> None:
 
         elif msg['type'] == RPCMessageType.SELL_NOTIFICATION:
             msg['amount'] = round(msg['amount'], 8)
-            msg['profit_percent'] = round(msg['profit_percent'] * 100, 2)
+            msg['profit_percent'] = round(msg['profit_ratio'] * 100, 2)
```

Comment:
Why change it on the right hand side, but not on the left? while it's passed to the message below - it's not visible to the user after it's converted to the message...

Reply:
yes, because `msg['profit_percent'] = `msg['profit_ratio'] * 100` here...